web/modules/custom/hoeringsportal_public_meeting/templates/hoeringsportal-public-meeting-summary.html.twig

```diff
@@ -90,7 +89,7 @@
   {% else %}
     {% if registration_deadline_passed %}
       <div class="">{{ 'Registration deadline passed'|t }}</div>
-    {% elseif availability.available_number != availability.total_size %}
+    {% elseif 0 == availability.available_number %}
```

Comment:
This might be just something i dont understand, but should this be a [sameas](https://twig.symfony.com/doc/3.x/tests/sameas.html)?